hypershift-operator/controllers/hostedcluster/internal/platform/openstack/openstack.go

```diff
@@ -77,7 +77,7 @@ func reconcileOpenStackClusterSpec(hcluster *hyperv1.HostedCluster, openStackClu
 	// Note: The default is required for now because there is no CLI option to set the MachineNetwork.
 	// See https://github.com/openshift/hypershift/pull/4287
 	if len(hcluster.Spec.Networking.MachineNetwork) == 0 {
-		machineNetworks = []hyperv1.MachineNetworkEntry{{CIDR: *ipnet.MustParseCIDR(defaultCIDRBlock)}}
+		machineNetworks = []hyperv1.MachineNetworkEntry{{CIDR: *ipnet.MustParseCIDR(openstackutil.DefaultMachineNetwork)}}
```

Comment:
Didn't we already default this elsewhere? Seems dangerous to do it twice.

Reply:
Right, we do it in the CLI when creating the cluster. I'll delete it here then.

Reply:
Hmm. I wonder if it would be safer to do it here The question is: do we want the defaulting behaviour to be the same when you're using the CLI as when you're not using the CLI? Probably?

Reply:
* CLI: this PR will provide a default
* non-CLI: we assume that HostedCluster's YAML has it and if not we'll return an error.

I don't think it'll be accepted to provide a default directly in the API.

What do you think?